protos/dcs/coalition/v0/coalition.proto

```diff
@@ -43,7 +43,7 @@ message AddGroupRequest {
     bool late_activation = 3;
     string name = 4;
     dcs.common.v0.Position position = 5;
-    repeated Point route = 6;
+    repeated Point waypoints = 6;
```

Comment:
This isn't renamed due to a linting warning is it? If not can we remove from this PR so that it is limited to linting fixes only.

Reply:
> This isn't renamed due to a linting warning is it?

The linter is throwing an error because "route" is not a plural.

If `waypoints` is not a suitable substitute, I would recommend the alternatives of:
* Support `route.points` where `route` will only have one property called `points` (the inspiration of the waypoints)
* Alternatively, disable the linter on this line -- subject to the review as we are developing this API.

Reply:
Thanks for the clarification. Lets leave this as is for now then. I think having a `Route` with `Points` as you suggest may be the way to go in the future but we can deal with that when the time comes.